packages/upgradeable/package.json

```diff
@@ -56,6 +56,7 @@
     "@nomicfoundation/hardhat-network-helpers": "^1.0.9",
     "@nomiclabs/hardhat-ethers": "npm:hardhat-deploy-ethers@^0.3.0-beta.13",
     "@openzeppelin/hardhat-upgrades": "1.28.0",
+    "@openzeppelin/upgrades-core": "1.30.0",
```

Comment:
Peer and dependencies? Maybe it must be in peer and dev dependencies?

Reply:
no, microbundle will inject it to the output bundle, that is not a desire behaviour